eloquentjs/20_2_directory_creation/src/modules/delete.ts

```diff
@@ -0,0 +1,34 @@
+import { IncomingMessage } from "node:http";
+import { stat, rmdir, unlink } from "node:fs/promises";
+import { urlPath } from "./helpers";
+
+
+const DELETE = async function(request: IncomingMessage) {
+    console.log("Recieved DELETE request");
+
+    const path = urlPath(request.url);
+
+    let stats;
```

Comment:
Just because a tool uses a non-descriptive name doesn't mean you need to. Try to figure out what this actually is and name it that. The same goes for all the other stats. Something I can't help but notice is that it doesn't seem to have anything to do with statistics, which is where my mind immediately went.